src/components/LocationMarker/__tests__/LocationMarker.test.js

```diff
@@ -0,0 +1,63 @@
+import L from 'leaflet';
```

Comment:
Any reason to use a one letter variable name here?

Reply:
That's how it's been done so far where Leaflet was included. I didn't think twice about it. I can change it, if you want me to.